doc/install.md

```diff
@@ -0,0 +1,58 @@
+# **Instructions for Preparing Raspberry Pi 5 for Integration with AosEdge**
+
+This guide covers writing the prepared image to an **SD card**, configuring the boot process, and the initial startup of **Raspberry Pi 5**.
+
+---
+
+## **1. Writing the Image to an SD Card**
+
+### **1.1 Download and Install Raspberry Pi Imager**
+
+1. Go to the official **[Raspberry Pi Imager](https://www.raspberrypi.com/software/)** website and download the software for your operating system.
+
+2. Install and launch Raspberry Pi Imager.
+
+### **1.2 Writing the Image to the SD Card**
+
+1. Insert the **SD card** into your computer.
+
+2. In **Raspberry Pi Imager**:
+
+   - Click **"Choose OS"** → **"Use custom"** and select your prepared image.
+
+   - Click **"Choose Storage"** and select your **SD card**.
+
+   - Click **"Write"** to start writing the image.
+
+3. Wait for the process to complete, then **safely remove** the SD card.
+
+---
+
+## **2. Assembling Raspberry Pi 5 and Connecting the NVMe SSD**
```

Comment:
We have two options: 
1. Use nvme hat
2. Use USB flash drive

Both options should be described.